### DISCUSSION TITLE
Hints popup after first dot printed

### DISCUSSION THREAD

**Discussion by 0x501D** [neoclide/coc.nvim#4173] [Q&A] [answered]
Hello. Is there any way to make popup menu works only after first dot entered.
E.g after prints `data.` ?

**Comment by chemzqm** [comment, marked as answer]
:h coc-suggest-autoTrigger

**Comment by 0x501D** [comment]
Tnanks!
```
"suggest.autoTrigger": "trigger"
```
Do the job.
